src/index.js

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { fb } from './lib/firebase';
+import { FirestoreProvider } from 'react-firestore';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(
+  <FirestoreProvider firebase={fb}>
```

Comment:
Well done! I like that you added the `FirestoreProvider` in this file so all parts of the app get to use the same provider. Organization ftw 👍